cookbooks/van-keken-vof/doc/van-keken-vof.tex

```diff
@@ -0,0 +1,216 @@
+within individual cells or within regions consisting of groups of cells.
+
+Another advantage the VOF method has over modeling moving interface problems with compositional 
+fields is that for problems in which the interface occupies a relatively small part of the 
+computational domain, all of the computational work in the VOF method is done in cells that lie 
+in a neighborhood of the interface, rather than in the entire computational domain. 
+
+\begin{figure}[b!]
+   \centering
+\includegraphics[width=0.5\textwidth]{cookbooks/van-keken-vof/doc/rms_vel_comparison.png}
+   \caption{Evolution of the root mean square velocity as a function of time for computations 
+    of the Van Keken problem made with the VOF interface tracking algorithm with five 
+    different global mesh refinements.
+    Since the VOF initial conditions are discontinuous, the above results should really be 
+    compared with the computations with discontinuous initial conditions on the left in 
+    Figure~\ref{fig:vk-2}. However, the above results also compare extremely favorably with the 
+    computations with smoothed, continuous initial conditions for the compositional field on 
+    the right in Figure~\ref{fig:vk-2}.
+    As in Figure~\ref{fig:vk-2}, 5 global refinements correspond to a $32 \times 32$ mesh and 10 global refinements correspond to a $1024 \times 1024$ mesh.}
+    \label{fig:vof-vk-1}
+\end{figure}
+
+\begin{figure}
+    \centering
+    \includegraphics[width=0.4\textwidth]{cookbooks/van-keken-vof/doc/vof_van_keken_refinement_comparison.png}
+    \caption{\it The results of two computations of the Van Keken problem made with the VOF 
+        interface tracking algorithm overlaid upon each other at $t_{end}=2000$.
+        This visualization shows the reconstructed boundary between the two
+        materials at the final time $t_{end}$ as computed on a uniform grid
+        with 7 and 8 levels of refinement.
+        The boundaries between the materials are displayed as contours
+        of the fields $\tilde{\psi}^7\,(t_{end})$ (black) and
+        $\tilde{\psi}^8\,(t_{end})$ (bright green), which are generated by the  visualization postprocessor.
+        The contours for the reconstructed material boundaries are superimposed
+        on a color gradient visualization of the material composition for the
+        computation with 8 levels of refinement in order to make the regions
+        with each fluid type more evident.
+        Compare with the fourth image on the right in Figure~\ref{fig:vk-1}.
+    }
+    \label{fig:VOF_van_Keken-02}
+\end{figure}
+
+Since, as noted above, when the interface is discontinuous, the van Keken problem is a version 
+of the Rayleigh-Taylor problem that is unstable to perturbations of all 
+wavelengths,\footnote{This is true whether the two fluids have the same viscosity or different 
+viscosities.} (e.g. see~\cite{SC:1961}).
+Therefore, it is extremely sensitive to the initial conditions.
```

Comment:
```suggestion
wavelengths\footnote{This is true whether the two fluids have the same viscosity or different 
viscosities.} (e.g. see~\cite{SC:1961}),
it is extremely sensitive to the initial conditions.
```

Reply:
@gassmoeller @naliboff Making a separate PR for the AMR computation and figure is fine with me.  

However, I have another question.  I have spotted one or more minor errors in **manual.bib**.  For example, in this reference

> [72] J. Ritsema and H. J. van Heijst. Seismic imaging of structural heterogeneity in earth’s mantle: ...

Earth is not capitalized here, since there needs to be curly braces around the E: {E}arth in the **manual.bib** file.

Shall I suggest the change here?  One thing to consider is that once I am done with this PR I am not likely to look at the manual or **manual.bib** for quite a while.  I have a lot on my plate at the moment.

Reply:
@gassmoeller Do you feel strongly about removing the complete sentence

> Therefore, it is extremely sensitive to the initial conditions.

and just adding

> ... it is extremely sensitive to the initial conditions.

to the end  of the previous sentence?  In other words, removing the word "Therefore, ..." as the beginning of the (complete) sentence quoted above.

I wrote those two sentences and I would prefer to remove the comma after **wavelengths** (it was definitely a mistake) but keep the period '.' after the right parentheses like this:

> (e.g. see~\cite{SC:1961}).

I think this is just a matter of writing style. :-)

Reply:
I have modified the text as suggested in @egpuckett's comment, and would appreciate a response on whether I should look into further modifications. I will note that to me the phrasing suggested by EGP is more natural, as it includes the transitioning structure. Some rephrasing might be appropriate, but just removing the word "Therefore" doesn't feel correct.

Reply:
I am perfectly fine with your solution, I just wanted to fix the additional comma in some form.